scripts/docker-build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e  # Exit on error.
+set -x  # Make command execution verbose
+
+function build_nikss() {
+  git submodule update --init --recursive
+  ./build_libbpf.sh
+  if [ -e build ]; then /bin/rm -rf build; fi
+  mkdir -p build
+  cd build
+  cmake "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}" ..
+  make -j$(nproc)
+  make install
+}
+
+function build_bpftool() {
+  git clone --recurse-submodules https://github.com/libbpf/bpftool.git /tmp/bpftool
```

Comment:
This build of bpftool is independent from kernel version?

Reply:
It looks so. We have never built bpftool for a specific kernel version. The bpftool repo doesn't say anything about it too.